### PR TITLE
Mssw chapter11

### PR DESCRIPTION
**Pull request by Spinodal**

### CODE REVIEW

File: chapter11/mssw/OogaboogooCalendar.pm [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package OogaboogooCalendar;
+
+use strict;
+use warnings;
+
+use Data::Dumper;
+
+use constant {
+    DAY => [ qw( ark dip wap sen pop sep kir ) ],
+    MONTH => [ qw( diz pod bod rod sip wax lin sen kun fiz nap dep ) ],
```
Comment (utopia-2) [inline chapter11/mssw/OogaboogooCalendar.pm:10]: Obligatory _not actually constant constants_ comment.

Comment (Spinodal) [inline chapter11/mssw/OogaboogooCalendar.pm:10]: Yea but it's the thought that counts :octopus:

Comment (utopia-2) [inline chapter11/mssw/OogaboogooCalendar.pm:10]: You won me over with the octopus. :shipit:

File: chapter11/mssw/exercise2.pl [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+use strict;
+use warnings;
+
+use Time::Piece;
+use OogaboogooCalendar;
+
+use Data::Dumper;
+
+my $t = localtime;
```
Comment (alexdeas) [inline chapter11/mssw/exercise2.pl:11]: Time::Piece seems to be mssw's most favourite thing this week.

File: chapter11/mssw/OogaboogooCalendar.pm [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+sub number_to_month {
+    my $num = shift;
+    if ($num > 11 || $num < 0 ){
+        die "Number provided not in range of months (0 - 11)";
+    }
+    return MONTH->[$num];
+}
+
+
+'blarg';
```
Comment (alexdeas) [inline chapter11/mssw/OogaboogooCalendar.pm:30]: Debatable whether this is actually true.

File: chapter11/mssw/OogaboogooCalendar.pm (hunk 1 above)
Comment (ardavey) [inline chapter11/mssw/OogaboogooCalendar.pm:10]:
``` perl
#!/usr/bin/perl
use strict;
use warnings;

use constant { FOO => [ 'my ', 'constants ' ] };
print FOO->[0], FOO->[1];
FOO->[0] = 'are not ';
print FOO->[0], FOO->[1];
```

File: chapter11/mssw/exercise2.pl [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+my $day = OogaboogooCalendar::number_to_day( $t->_wday);
+my $month = OogaboogooCalendar::number_to_month( $t->_mon);
+
+printf( "Today is %s %u %s %u\n", $day, $t->mday, $month, $t->year );
```
Comment (ardavey) [inline chapter11/mssw/exercise2.pl:16]: Well, just look at `%u`!

**Pull request merged**